TACC/Core-Styles
# fix: form elements selectors too specific

## Overview

Remove specificity of form elements, so clients can easily overwrite their complex selectors.

## Related

- fixes bug in https://github.com/TACC/tup-ui/pull/270

## Changes

- add `:where()` to form element selectors using only `:is()` or `:not()`

## Testing

1. Verify checkbox labels are inline with checkbox.
2. Verify multi-radio & multi-checkbox inputs' labels are inline with box.
3. ~~Verify fieldsets have margin `3.5rem` between them.~~*

<sup>* Our forms do not use fieldsets, so testing is neglected right now.</sup>

## UI

<img width="1200" alt="Screenshot 2023-08-02 at 10 26 27 AM" src="https://github.com/TACC/Core-Styles/assets/62723358/240bed59-9325-46ca-adaf-088dba52cbb2">
